qiskit/opflow/primitive_ops/tapered_pauli_sum_op.py

```diff
@@ -391,7 +405,9 @@ def _taper(self, op: PauliSumOp, curr_tapering_values: List[int]) -> OperatorBas
             z_temp = np.delete(pauli_term.primitive.paulis.z[0].copy(), np.asarray(self._sq_list))
             x_temp = np.delete(pauli_term.primitive.paulis.x[0].copy(), np.asarray(self._sq_list))
             pauli_list.append((Pauli((z_temp, x_temp)).to_label(), coeff_out))
-        spo = SparsePauliOp.from_list(pauli_list).simplify(atol=0.0)
+
+        spo = SparsePauliOp.from_list(pauli_list).simplify(0.0)
```

Comment:
```suggestion
        spo = SparsePauliOp.from_list(pauli_list).simplify(atol=0.0)
```
I think it's clearer to keep this.